src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/IVsTelemetryService.cs

```diff
@@ -7,14 +7,9 @@ namespace Microsoft.VisualStudio.Telemetry
     /// <summary>
     /// Internal as we don't want anyone who depends on us to post events to our eventId.
     /// </summary>
-    internal interface ITelemetryService
+    internal interface IVsTelemetryService
     {
         /// <summary>
-        /// Posts a given telemetry event path to the telemetry service session for the program.
-        /// </summary>
-        void PostEvent(string eventName);
-
-        /// <summary>
         /// Posts a given telemetry operation to the telemetry service. <seealso cref="TelemetrySessionExtensions.PostOperation(TelemetrySession, string, TelemetryResult, string, TelemetryEventCorrelation[])"/>
         /// </summary>
         /// <param name="eventName">Name of the event.</param>
```

Comment:
For the Event itself to be created(or any of the derived TelemetryEvent type), we will need to set some enums which we can handle in the non-VS layer using some Mapped Enum. So, when we have a need to add a different type of events, we can add more APIs to ITelemetryService and report the events. 

We will need the IVSTelemetryService only for some advanced scenarios. I dont know what they are. For that matter, nobody consumes the APIs on the VS layer except for some tests. We could as well remove it and add it back when someone needs it. 

Now that I think about it, we should remove IVSTelemetryService and add it when someone needs it.